backstop/tests/select.html

```diff
@@ -72,7 +73,7 @@ <h1>Select</h1>
     <h2>Default</h2>
     <section id="demo-default">
 
-      <section id="demo-default-singular">
+      <div id="demo-default-singular">
```

Comment:
so if not needed, remove this and pass div on label maybe?

Reply:
Derp, that makes much more sense.  ID is moved to the label.